Commands/utils.py

```
@@ -0,0 +1,212 @@
+        "stop": list or None,
+        "system_message": str,
+        "grounded": bool,
+        "grounding_text": str or list[str]
+    }
+    client: API client object
+    Returns generated text.
+    """
+    try:
+        prompt = request.get("prompt", "")
+        grounded = request.get("grounded", False)
+        grounding_text = request.get("grounding_text", None)
```

Ruff (0.11.9)

Use `request.get("grounding_text")` instead of `request.get("grounding_text", None)`

Replace `request.get("grounding_text", None)` with `request.get("grounding_text")`

(SIM910)